src/PrestaShopBundle/Controller/Admin/CommonController.php

```diff
@@ -147,7 +147,7 @@ public function paginationAction(Request $request, $limit = 10, $offset = 0, $to
             'limit_choices' => $limitChoices,
         );
         if ($view != 'full') {
-            return $this->render('PrestaShopBundle:Admin:Common/pagination_'.$view.'.html.twig', $vars);
+            return $this->render('PrestaShopBundle:Admin/Common/pagination_'.$view.'.html.twig', $vars);
```

Comment:
should be done in another pull request (pagination in 'grid' view don't works as expected).